src/Mod/Draft/draftmake/make_label.py

```diff
@@ -280,9 +282,13 @@ def make_label(target_point=App.Vector(0, 0, 0),
     if not custom_text:
         custom_text = "Label"
     try:
-        utils.type_check([(custom_text, str)], name=_name)
+        utils.type_check([(custom_text, (str, list))])
```

Comment:
Why does this remove the `name` argument? That doesn't seem related to the main change, and it looks like passing a name here is still relevant? If it is somehow not possible with the main change, then IMHO at least the commit message should document why it must be removed. If this is just because the `make_text` function doesn't pass the argument, then it should probably be kept here (and maybe added to `make_text` even).

Reply:
My bad. I will fix that.

Reply:
FYI the mentioned name argument is used for messages in the Report view:
`typecheck[make_label]: '1' is not (<class 'str'>, <class 'list'>)`